openvdb/points/AttributeArrayString.h

```diff
@@ -25,20 +25,20 @@ namespace points {
 ////////////////////////////////////////
 
 
-using StringIndexType = uint32_t;
+using StringIndexType = Index;
```

Comment:
Why change this then remove its usage below and in the .cc?

Reply:
I thought the code was simpler using the Index typedef directly as I'm not sure we will ever change this. I didn't want to remove it directly, but I realize I can mark this deprecated:

`using StringIndexType OPENVDB_DEPRECATED = Index;`

?

Reply:
Edit - Turns out this doesn't work on Windows, so had to remove the deprecation there.